service/internal/customizations/accept_encoding_gzip_test.go

```diff
@@ -196,3 +196,20 @@ func (*stubOpDeserializer) HandleDeserialize(
 ) {
 	return next.HandleDeserialize(ctx, input)
 }
+
```

Comment:
What do you think about  putting this under a more specific customization package so it doesn't get lost. or becomes a dumping area.  maybe something like `service/internal/customizations/accept-encoding-gzip`.

Also the `dynamodb/internal/customization/doc.go` might need a tweak to to say it refers to this customization maybe?

Reply:
moved to `service/internal/accept-encoding`. Also made the accept-encoding a separate go mod